build.gradle

```diff
@@ -262,6 +383,7 @@ project(':functional') {
     dependencies {
         compile "junit:junit:4.13"
         compile project(':api')
+        compile project(':minio-admin')
     }
```

Comment:
Since this is already `minio-java` does it make sense to just call as this `:admin` or `:admin-api` ?  @balamurugana

Reply:
@harshavardhana Makes sense. We could just have `admin`

Reply:
Agreed, I think that's a lot cleaner, will change to `admin`.